src/Resources/views/CRUD/Association/edit_many_to_one.html.twig

```diff
@@ -24,11 +24,11 @@ file that was distributed with this source code.
                         'code':     sonata_admin.field_description.associationadmin.code,
                         'objectId': sonata_admin.field_description.associationadmin.id(sonata_admin.value),
                         'uniqid':   sonata_admin.field_description.associationadmin.uniqid,
-                        'linkParameters': sonata_admin.field_description.option('link_parameters')
+                        'linkParameters': sonata_admin.field_description.option('link_parameters', {})
                     })) }}
-                {% elseif sonata_admin.field_description.option('placeholder') %}
+                {% elseif sonata_admin.field_description.option('placeholder', 'short_object_description_placeholder') %}
```

Comment:
this can't be correct, otherwise it is an always true if and can be removed with just else

Reply:
To disabled the placeholder, 'false' should be use.